src/tasks/deploy.mk

```diff
@@ -102,6 +102,7 @@ change-api:
 			}, \
 			\"environment\": \"production\", \
 			\"systemCode\": \"$(CIRCLE_PROJECT_REPONAME)\", \
```

Comment:
Surely systemCode is not reliably the same as repo name?

Reply:
No it isn't. But we don't have the system code available in the builds. Hurts us for projects like next-es-interface and next-service-registry where it doesn't match.

We _could_ look it up from https://next-registry.ft.com/v2/.

Reply:
@nl-ria @dora - will anything break if the system code is not correct/missing

Probably best to leave blank if it's not accurate in any case

Reply:
🤔 It _shouldnt break_ anything if the `systemCode` isnt found in Biz-Ops. The only information you lose is the conversion of `systemCode` to the _system name_ and the link to the runbooks. e.g. 
![image](https://user-images.githubusercontent.com/5927420/61954846-b3ac5a80-afb1-11e9-8637-a30e956dfe25.png)
^ this link take you to the runbooks
vs 

![image](https://user-images.githubusercontent.com/5927420/61954939-e8201680-afb1-11e9-8a76-af418b5000da.png)
 ^ This link will `404`

They are both adding the GH link if a commit hash is provided. 

I would not recommend leaving `systemCode` blank, since it is a mandatory field for Change API. How about using the Biz-Ops API [merge](https://github.com/Financial-Times/biz-ops-api/blob/master/ENDPOINTS.md#merge) endpoint to make the `systemcode` the same as the repo name?

In the interim,  let me try and figure out why some of them dont have the GH links. Hold this space please.

Reply:
> I would not recommend leaving systemCode blank, since it is a mandatory field for Change API. How about using the Biz-Ops API merge endpoint to make the systemcode the same as the repo name?

Oh, that is a thing! Nice yes we'll make that happen, there are not too many systems.

Reply:
good idea, though I worry about future proofing (consider the current grafana metric naming issues you have because there's nothing to enforce choosing consistent names for things)

> I would not recommend leaving systemCode blank

But I'd argue its dangerous to deliberately send incorrect data in. Makes it harder to correlate changes to other system code indexed stuff.

Could one of your amazing github tools set a SYSTEM_CODE variable in all your makefiles or project vault folders @sjparkinson, and use that?

Reply:
I have made a change that looks up the correct system code from our service registry document.

Reply:
🙌